packages/google_maps_flutter/google_maps_flutter_android/README.md

```diff
@@ -70,8 +70,14 @@ AndroidMapRenderer mapRenderer = AndroidMapRenderer.platformDefault;
   }
 ```
 
-Available values are `AndroidMapRenderer.latest`, `AndroidMapRenderer.legacy`, `AndroidMapRenderer.platformDefault`.
-Note that getting the requested renderer as a response is not guaranteed.
+NOTE: `AndroidMapRenderer.platformDefault` corresponds to `AndroidMapRenderer.latest`.
```

Comment:
nit: Note is specfifically called out as an anti pattern https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose

Reply:
Done.